rusk/src/lib/http/prover.rs

```diff
@@ -20,7 +21,7 @@ impl HandleRequest for LocalProver {
     ) -> anyhow::Result<ResponseData> {
         let topic = request.event.topic.as_str();
         let response = match topic {
-            "prove_execute" => self.prove_execute(request.event_data())?,
+            "prove_tx_circuit" => LocalProver::prove(request.event_data())?,
```

Comment:
I agree with this change, however this is a breaking change that would need a version update.

I suggest to keep the old topic to be backward compatible.

As soon as we move to RUES dispatch event we can choose the name we prefer, since it will be a breaking change anyway